flux/src/helm-releases/HelmReleaseList.tsx

```diff
@@ -45,9 +55,18 @@ function HelmReleasesListWrapper() {
     return helmReleases?.makeCRClass();
   }, [helmReleases]);
 
+  const isFluxInstalled = useFluxInstallCheck();
```

Comment:
not critical but this repeating part could be extracted to a separate component that can be used to wrap page components

then in index.ts it would look something like this: 
```tsx
registerRoute({
  path: '/flux/helmreleases',
  parent: 'flux',
  sidebar: 'helmreleases',
  component: () => <FluxInstalledOnly><HelmReleases /></FluxInstalledOnly>,
  exact: true,
  name: 'helmreleases',
});

Reply:
Maybe for later.